test/commands/api/update.test.js

```diff
@@ -186,8 +201,8 @@ describe('valid api:update', () => {
   test
     .stub(config, 'getConfig', () => ({ SWAGGERHUB_URL: shubUrl }))
     .nock(`${shubUrl}/apis`, api => api
-      .get('/org/api/2.0.0')
-      .reply(200)
+      .get('/org/api/2.0.0/settings/version')
```

Comment:
We see that by passing `visibility=public` we can override the current true value to make it false.